contracts/gas-snapshots/functions.gas-snapshot

```diff
@@ -53,23 +53,23 @@ FunctionsRouter_ProposeContractsUpdate:test_ProposeContractsUpdate_RevertIfNotNe
 FunctionsRouter_ProposeContractsUpdate:test_ProposeContractsUpdate_RevertIfNotOwner() (gas: 23369)
 FunctionsRouter_ProposeContractsUpdate:test_ProposeContractsUpdate_Success() (gas: 118456)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfConsumerNotAllowed() (gas: 59304)
-FunctionsRouter_SendRequest:test_SendRequest_RevertIfDuplicateRequestId() (gas: 192478)
+FunctionsRouter_SendRequest:test_SendRequest_RevertIfDuplicateRequestId() (gas: 192796)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfEmptyData() (gas: 29405)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfIncorrectDonId() (gas: 57926)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfInsufficientSubscriptionBalance() (gas: 186209)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfInvalidCallbackGasLimit() (gas: 50902)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfInvalidDonId() (gas: 25061)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfNoSubscription() (gas: 29111)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfPaused() (gas: 34247)
-FunctionsRouter_SendRequest:test_SendRequest_Success() (gas: 198072)
+FunctionsRouter_SendRequest:test_SendRequest_Success() (gas: 284999)
```

Comment:
I know a lot of this additional gas is because of the test changes, but I am curious how much more gas it adds to hash all this data. Anyone have some rough numbers?